static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -325,67 +310,28 @@ function FlamegraphZoomView({
     flamegraphSearch,
   ]);
 
+  const selectedFramesRef = useRef<FlamegraphFrame[] | null>(null);
   useEffect(() => {
     if (!flamegraphCanvas || !flamegraphView || !selectedFrameRenderer) {
       return undefined;
     }
 
-    const drawFocusedFrameBorder = () => {
-      selectedFrameRenderer.draw(
-        focusedRects,
-        {
-          BORDER_COLOR: flamegraphTheme.COLORS.FOCUSED_FRAME_BORDER_COLOR,
-          BORDER_WIDTH: flamegraphTheme.SIZES.FOCUSED_FRAME_BORDER_WIDTH,
-        },
-        flamegraphView.fromConfigView(flamegraphCanvas.physicalSpace)
-      );
-    };
-
-    scheduler.registerAfterFrameCallback(drawFocusedFrameBorder);
-    scheduler.draw();
-
-    return () => {
-      scheduler.unregisterAfterFrameCallback(drawFocusedFrameBorder);
-    };
```

Comment:
This doesn't initialize the `selectedFramesRef` on load anymore which breaks the highlighting function from the suspect functions link.

Reply:
Oh I made a mistake merging this. Let me revert it 🤦🏼